packages/account/src/utils/Types.ts

```diff
@@ -177,6 +177,20 @@ export type InitilizationData = {
   signerAddress?: string;
 };
 
+export type PaymasterUserOperationDto = {
+  mode: PaymasterMode;
+  calculateGasLimits?: boolean;
+  expiryDuration?: number;
+  webhookData?: {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    [key: string]: any;
+  };
+  smartAccountInfo?: SmartAccountData;
+  feeTokenAddress?: string;
+  tokenList?: string[];
+  preferredToken?: string;
```

Comment:
Hex?

Reply:
Can leave it as string for now, if I change to hex will probably need to check other places as well.